docs/model_server_c_api.md

```diff
@@ -0,0 +1,45 @@
+# OpenVINO Model Server C-API (preview feature) {#ovms_docs_c_api}
+
+## Introduction
+
+This document describes OpenVINO Model Server (OVMS) C API that allows OVMS to be linked into C/C++ applications. With exceptions listed below all capabilities of OVMS are included in the shared library.
+
+## Preview limitations
+* Launching server in single model mode is not supported. You must use configuration file.
+* DAG pipelines cannot be used directly through C API. Inferences can be scheduled only by gRPC/HTTP endpoints.
```

Comment:
```suggestion
* DAG pipelines cannot be used directly through C API. DAG inferences can be scheduled only by gRPC/HTTP endpoints.
```

Otherwise it sounds like ALL inferences can be scheduled only via network